utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -205,7 +205,7 @@ private fun ArchiveInputStream.unpack(
 
             val target = targetDirectory.resolve(entry.name)
 
-            // There is no guarantee that directory entries appear before file entries, so always ensure the parent
+            // There is no guarantee that directory entries appear before file entries, so ensure that the parent
```

Comment:
Actually, the "always" was intended to emphasize we unconditionally do this here instead of e.g. inside an `if (entry.isDirectory) {}` block. Anyway, if you want to reword this comment, please also reword the other two occurrences of this comment in this file.

Reply:
If adjusted the other ocurences.